Shopify/android-testify
# Fix issue #130 - Detect and warn if Activity is not registered in the manifest

### What this does

Resolves #130 

It was a common error to omit adding test harness Activities to the AndroidManifest of your androidTest configuration. When this happened, a cryptic error message about being unable to launch the Activity was thrown. Instead, we now catch this error and present a more actionable error in an effort to guide the user towards fixing the issue easily.

### How to test

You can remove the AndroidManifest entry for one of the TestHarnessActivities in the Sample app and attempt to run the tests.
Or, there is a new instrumentation test to check this condition in `ActivityNotRegisteredExceptionTest`
